atomate/common/firetasks/glue_tasks.py

```diff
@@ -73,6 +73,94 @@ def get_calc_loc(target_name, calc_locs):
         return calc_locs[-1]
 
 
+@explicit_serialize
+class GrabFilesFromCalcLoc(FiretaskBase):
+    """
+    Based on CopyVaspOutputs but for general file copying.
+
+    Required params:
+        filenames (str): filenames to copy.
```

Comment:
Looks like filenames can be a str or list or $ALL.

I'd actually just make it a list or $ALL (and remove single file string option) so that the argument is a bit more specific, but it's up to you.